tests/StatsdClient.Tests/StatsdUnitTests.cs

```diff
@@ -21,206 +21,204 @@ public void Setup()
             _stopwatch = Mock.Of<IStopWatchFactory>();
         }
 
-
         // =-=-=-=- COUNTER -=-=-=-=
 
         [Test]
-        public void send_increase_counter_by_x()
+        public void Send_increase_counter_by_x()

```

Comment:
we use CamelCase almost everywhere, should we consider dropping the snake_case and renaming these?

Reply:
Ideally we should use CamelCase everywhere. I didn't do it as there are 211 functions to rename (I used the `Replace All` feature to capitalize the first letter).
Do you think I should rename these functions?

Reply:
Let's rename those in a separate PR. I have the same issue with some java tests.